src/components/crossDevice/Intro/index.js

```diff
@@ -0,0 +1,46 @@
+import { h } from 'preact'
+import classNames from 'classnames'
+import theme from '../../Theme/style.css'
+import style from './style.css'
+import Title from '../../Title'
+import { trackComponent } from '../../../Tracker'
+import {preventDefaultOnClick} from '../../utils'
+
+const Intro = ({i18n, nextStep, componentsList, crossDeviceInitialStepType}) => {
+  const isFace = crossDeviceInitialStepType === 'face'
+  const steps = {
+    'sms': 'sms',
+    'take-photos': `${ isFace ? 'face' : 'document' }.take_photos`,
+    'return-computer': 'return_computer',
+  }
+  return (
+    <div className={classNames(theme.fullHeightContainer, style.container)}>
+      <Title
+        title={i18n.t(`cross_device.intro.${ isFace ? 'face' : 'document' }.title`)}
+        subTitle={i18n.t('cross_device.intro.sub_title')}
+      />
+      <div className={classNames(theme.thickWrapper, style.content)}>
+      {
+        Object.keys(steps).map(key =>
```

Comment:
Minor: are we not using an object mapper some where?

Reply:
I don't think so, but would be happy use an object mapper and add it to utils if you think it's useful.